Sockets/BackEndMonitoringSockets/include/CServiceConnectionHandler.h

```diff
@@ -18,14 +18,12 @@ class CServiceConnectionHandler
 	bool HandleEvent(const int socket_fd, EEventType type);
 
 private:
-	const int c_client_disconnected = 10054;
-
 	bool HandleRequestEvent(const int socket_fd);
 	bool HandleResponseEvent(const int socket_fd, EClientRequestType type);
-	bool HandleResponseExitEvent(const int socket_fd);
+	bool HandleWrongRequestEvent(const int socket_fd);
+	bool IsEqualStrings(const std::string first_str, const std::string second_str) const;
```

Comment:
I do not this that ServiceConnectionHandler should compare strings, standard methods do it better, also if you passing a const string to the function/method pass it by ref

Reply:
I added ref but for some reason, simple compearing didn't work. So I used comparing symbol by symbol and it helped.